blossom/templates/app/partials/tutorial.partial

```diff
@@ -1,37 +1,356 @@
-<div class="modal fade" id="tutorialModal" aria-hidden="true" aria-labelledby="exampleModalToggleLabel"
+{% load static %}
+
+<div class="modal fade" id="tutorialModal" aria-hidden="true" aria-labelledby="tutModalToggleLabel"
```

Comment:
This file is huge, can we put each modal in a separate file to make it easier to understand?